src/main/groovy/com/github/jk1/license/filter/LicenseBundleNormalizer.groovy

```diff
@@ -167,72 +173,81 @@ class LicenseBundleNormalizer implements DependencyFilter {
         importedModuleBundle.modules.addAll(normalizedModuleData)
     }
 
-    private Collection<License> normalizePomLicense(License license) {
-        List<NormalizerTransformationRule> rules = [ ]
+    @CompileStatic
+    private Collection<License> normalizePomLicense(License license,
+                                                    String module) {
+        List<NormalizerTransformationRule> rules = transformationRulesFor(module, license.name, license.url)
 
-        rules += findMatchingRulesForName(license.name)
-        rules += findMatchingRulesForUrl(license.url)
         LOGGER.debug("License {} ({}) matches the following rules: [{}]", license.name, license.url, rules.join(","))
 
         if (rules.isEmpty()) return [license]
 
         rules.collect { normalizePomLicense(it, license) }
     }
-    private Collection<ManifestData> normalizeManifestLicense(ManifestData manifest) {
-        List<NormalizerTransformationRule> rules = [ ]
 
-        rules += findMatchingRulesForName(manifest.license)
-        rules += findMatchingRulesForUrl(manifest.license)
+    @CompileStatic
+    private Collection<ManifestData> normalizeManifestLicense(ManifestData manifest,
+                                                              String module) {
+        List<NormalizerTransformationRule> rules = transformationRulesFor(module, null, manifest.license)
```

Comment:
When the manifest contains license information, usually the field `license` is used. Problem with that field is, that it's not clear what information is in there. Sometimes it is the license name, sometimes it's the license URL.
This change now eliminates the check for license name.

Reply:
Oh, right. It worked for some of "my" licenses.
But yes, you're right.
(License matching from POMs is a pain)